inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md

```diff
@@ -56,6 +56,30 @@ npm run build
 ```
 If you did everything right, you'll get `✔ Successfully compiled the project.` in the terminal.
 
+<br/>
+
+Please note that before Paraglide-js's Vite plugin is released, you may be troubled by [Vite's Dependency Pre-Bundling](https://vitejs.dev/guide/dep-pre-bundling.html), which saves compiled dependencies in `node_modules/.vite` for subsequent use. However, the working mechanism of `paraglide-js` causes Vite to incorrectly cache `messages.js`, causing the application to use stale messages.
+
+We recommend that you add `@inlang/paraglide-js` to the exclusion list of dependency optimizations:
+```js
+// vite.config.js/ts
+export default defineConfig({
+  optimizeDeps: {
+    exclude: ['@inlang/paraglide-js']
+  },
+});
+```
+
+Or just in local environment (`serve` or `dev` command):
+```js
+// vite.config.js/ts
+export default defineConfig(({ command }) => ({
+  optimizeDeps: {
+    exclude: command === 'serve' ? ['@inlang/paraglide-js'] : []
+  },
+}));
+```
```

Comment:
I don't think this is necessary. 

The vite docs mention that optimizeDeps only happens in dev mode. "Dependency pre-bundling only applies in development mode".

Reply:
you are right! I'll remove this part.